src/style.css

```
@@ -1 +1,75 @@
-/* add styles here */
+/* #region body */
+
+body {
+  margin: 0;
+  background-color: #fff;
+}
+
+/* #endregion */
+
+/* #region container */
+
+/* .container {
+  box-sizing: border-box;
+  width: 140px;
+  height: 160px;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+} */
+
+/* #endregion */
+
+/* #region stars */
+
+.stars--0 {
```

```suggestion
.stars {
```
You can set the same styles for the basic class and reuse this class in your HTML